packages/bun-types/bun.d.ts

```diff
@@ -2860,6 +2860,13 @@ declare module "bun" {
   // tslint:disable-next-line:unified-signatures
   function file(path: string | URL, options?: BlobPropertyBag): BunFile;
 
+  /**
+   * A list of files embedded into the standalone executable. Lexigraphically sorted by name.
+   *
+   * If the process is not a standalone executable, this returns an empty array.
+   */
+  const embeddedFiles: ReadonlyArray<Blob>;
```

Comment:
a next step would to add a easy way to search/index files like

```js
const sprite = Bun.Dir(emdeddedFiles).get("sprite.png")
```

probably everyone will just import and use from the import instead of using a list of embeddedFiles